docs/content/en/docs/getting-started/site-engineer/_index.md

```diff
@@ -0,0 +1,232 @@
+
+{{% alert title="Tip" color="success" %}}
+Kubernetes Ingress configuration is a complicated topic. Really all that is happening in this upgrade is that a set of external URIs and ports are being mapped to internal Kubernetes resources. It's not necessary to understand all of the configuration particulars in order to successfully complete this section.
+{{% /alert %}}
+
+To upgrade K8ssandra:
+
+1. Copy the following YAML into a new file named `k8ssandra-traefik.values.yaml`:
+
+    ```yaml
+    cassandra:
+      version: "3.11.7"
+      auth:
+        enabled: false
+      clusterName: k8ssandra
+      datacenters:
+      - name: dc1
+        size: 1
+    ingress:
+      traefik:
+        enabled: true
+        repair:
+          enabled: true
+          host: repair.127.0.0.1.nip.io
+        stargate:
+          enabled: true
+          host: stargate.127.0.0.1.nip.io
+    kube-prometheus-stack:
+      prometheus:
+        enabled: true
+        prometheusSpec:
+          externalUrl: http://localhost:9090/prometheus
+          routePrefix: /prometheus
+        ingress:
+          enabled: true
+          paths:
+            - /prometheus
+      grafana:
+        enabled: true
+        ingress:
+          enabled: true
+          path: /grafana
+        adminUser: admin
+        adminPassword: admin123
+        grafana.ini:
+          server:
+            root_url: http://localhost:3000/grafana
+           serve_from_sub_path: true
+    ```
+
+1. Copy the following YAML into a file named `traefik.values.yaml`:
+
+    ```yaml
+    ---
+    providers:
+      kubernetesCRD:
+        namespaces:
+          - default
+          - traefik
+      kubernetesIngress:
+        namespaces:
+          - default
+          - traefik
+
+    ports:
+      traefik:
+        expose: true
+        nodePort: 32090
+      web:
+        nodePort: 32080
+      websecure:
+        nodePort: 32443
+      cassandra:
+        expose: true
+        port: 9042
+        nodePort: 32091
+      cassandrasecure:
+        expose: true
+        port: 9142
+        nodePort: 32092
+      sg-graphql:
+        expose: true
+        port: 8080
+        nodePort: 30080
+      sg-auth:
+        expose: true
+        port: 8081
+        nodePort: 30081
+      sg-rest:
+        expose: true
+        port: 8082
+        nodePort: 30082
+
+    service:
+      type: NodePort
+    ```
+
+1. Install Traefik:
+
+    ```bash
+    helm install -f traefik.values.yaml traefik traefik/traefik -n traefik --create-namespace
+    NAME: traefik
+    LAST DEPLOYED: Fri Feb 19 12:40:36 2021
+    NAMESPACE: traefik
+    STATUS: deployed
+    REVISION: 1
+    TEST SUITE: None
+    ```
+
+1. Upgrade the K8ssandra pod with Traefik support:
```

Comment:
The "K8ssandra pod"? Shouldn't it say something like "K8ssandra installation" or "K8ssandra deployment"?

Reply:
Yes, duh.